docs/source/release_notes.rst

```diff
@@ -276,15 +394,17 @@ Several modules have reported UTF-8 decoding errors when interacting with result
 An undocumented option **size** was defined in module **zos_data_set**, this has been removed to satisfy collection certification, use the intended
 and documented **space_primary** option.
 
-In the past, choices could be defined in either lower or upper case. Now, only the case that is identified in the docs can be set,
-this is so that the collection can continue to maintain certified status.
-
-Availability
-------------
+    - Change action plugin call from copy to zos_copy.
+    - Previous code did not return output, but still requested job data from the target system. This changes to honor `return_output=false` by not querying the job dd segments at all.
+- ``zos_operator`` - Changed system to call `wait=true` parameter to zoau call. Requires zoau 1.2.5 or later.
+- ``zos_operator_action_query`` - Add a max delay of 5 seconds on each part of the operator_action_query. Requires zoau 1.2.5 or later.
+- ``zos_unarchive``
 
-* `Automation Hub`_
-* `Galaxy`_
-* `GitHub`_
+    - Add validation into path joins to detect unauthorized path traversals.
+    - Enhanced test cases to use test lines the same length of the record length.
+- ``module_utils/template`` - Add validation into path joins to detect unauthorized path traversals.
+- ``zos_tso_command`` - Add example for executing explicitly a REXX script from a data set.
+- ``zos_script`` - Add support for remote_tmp from the Ansible configuration to setup where temporary files will be created, replacing the module option tmp_path.
```

Comment:
the above 5 lines may be another merge error/issue

Reply:
Fixed the release notes